source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -580,6 +580,10 @@ def __init__(self):
         if handle_poi_infos(args["add_poi_infos"], args["bss_stands"]):
             self.get_decorators.insert(1, ManageParkingPlaces(self, 'journeys'))
 
+        parser_get.add_argument(
+            "max_waiting_duration", type=PositiveInteger(), help='Maximum waiting, must be > 0'
```

Comment:
```suggestion
            "max_waiting_duration", type=PositiveInteger(), 
            help='A journey containing a waiting section with a duration greater or equal to  max_waiting_duration will be discarded. Units : seconds. Must be > 0'
```
Just to be a bit more precise

Reply:
:+1: